docs/en/sql-reference/statements/show.md

```diff
@@ -651,3 +651,47 @@ If either `LIKE` or `ILIKE` clause is specified, the query returns a list of sys
 
 **See Also**
 - [system.functions](../../operations/system-tables/functions.md) table
+
+## SHOW MERGES
+
+Returns a list of merges. All merges are listed in the [system.merges](../../operations/system-tables/merges.md) table.
+
+
+**Syntax**
+
+``` sql
+SHOW MERGES [[NOT] LIKE|ILIKE '<table_name_pattern>'] [LIMIT <N>]
+```
+
+**Examples**
+
+Query:
+
+``` sql
+SHOW MERGES;
+```
+
+Result:
+
+```text
```

Comment:
The examples need to be updated with the output of the new query

Reply:
Updated.